.github/workflows/publish.yml

```diff
@@ -33,13 +33,10 @@ jobs:
              # we need to build our own version of cross-enabled linuxkit, while upstream is sorting things out
              make LINUXKIT_VERSION=master LINUXKIT_SOURCE=https://github.com/rvs/linuxkit.git linuxkit
              # constraining environment for riscv64 builds
-             EVE_BUILDER_IMAGE=$(./build-tools/bin/linuxkit pkg show-tag pkg/alpine)-riscv64
              echo "PKGS=pkg/alpine pkg/ipxe pkg/mkconf pkg/mkimage-iso-efi pkg/mkimage-raw-efi" >> "$GITHUB_ENV"
              echo "ZARCH=riscv64" >> "$GITHUB_ENV"
-             echo 'LK_BUILD_ARGS={"EVE_BUILDER_IMAGE": "'$EVE_BUILDER_IMAGE'"}' >> "$GITHUB_ENV"
+             echo 'LK_BUILD_ARGS={"EVE_BUILDER_IMAGE": "lfedge/eve-alpine:abcff574e06af277a13fe88c9f4c59407e1ba1ff-riscv64"}' >> "$GITHUB_ENV"
```

Comment:
Why are we hard-coding the individual hash into the workflow file? Everything else seems to use `lfedge/eve-alpine:6.2.0` in the Dockerfiles; could we just create a 6.3.0 (or 6.2.1), which is a multi-arch index, which also includes reference to the risc-v manifest?

Reply:
It does not include a reference to risc-v manifest. But the bigger problem is that we're still using a complete override (done by the hacks I introduced into linuxkit as I was explaining on the other PR) so right now it is unavoidable. However, my plan is: for 6.7.0 release -- we ARE going to produce eve-alpine that WILL include riscv64 portion and if by that time linuxkit is reasonable -- we can switch to that.

For now -- there's not really any better way -- but incrementally this PR at least reduced some amount of cruft (still more cruft to go -- to be sure!)

Ok if we merge it now @deitch ?